src/main/java/com/faforever/client/player/PlayerService.java

```diff
@@ -8,6 +8,8 @@
 
 public interface PlayerService {
 
+  Boolean isOnline(String username);
```

Comment:
`boolean` instead of `Boolean`, because one can be `null` while the other can't.